x/session/keeper/session_hydrator.go

```diff
@@ -9,11 +9,10 @@ import (
 
 	sdkerrors "cosmossdk.io/errors"
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	_ "golang.org/x/crypto/sha3"
-
 	"github.com/pokt-network/poktroll/x/session/types"
 	sharedhelpers "github.com/pokt-network/poktroll/x/shared/helpers"
 	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
+	_ "golang.org/x/crypto/sha3"
```

Comment:
https://github.com/pokt-network/poktroll/pull/253/files#r1423131221 again this should be the middle section